lib/govuk_navigation_helpers/navigation_sidebar.rb

```diff
@@ -0,0 +1,85 @@
+require 'govuk_navigation_helpers/services'
+require 'govuk_navigation_helpers/configuration'
+
+module GovukNavigationHelpers
+  class NavigationSidebar
+    def initialize(content_item)
+      @content_item = ContentItem.new content_item
+    end
+
+    def navigation_sidebar
+      {
+        related_items: related_items,
+        collections: related_collections,
```

Comment:
I think we need to test that related_items, collections, topics, policies and publishers all get dug out of the content item response correctly (even though their retrieval is very trivial) as it will guard against breakages if something changes.
This could just be one simple fair-weather test for them all.